apps/application-system/api/src/environments/environment.ts

```diff
@@ -25,4 +25,5 @@ export default {
   auth: {
     jwtSecret: 'supersecret',
   },
+  fsS3Bucket: process.env.FILE_SERVICE_PRESIGN_BUCKET,
```

Comment:
Is this always read from the proccess environment or is sometimes hardcoded for local development?